remixer_core/src/main/java/com/google/android/libraries/remixer/Remixer.java

```diff
@@ -16,13 +16,25 @@
 
 package com.google.android.libraries.remixer;
 
+import com.google.android.libraries.remixer.sync.SynchronizationMechanism;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Locale;
 
 /**
- * Contains a list of {@link Variable}es.
+ * Contains a list of {@link Variable}s or {@link Trigger}s.
+ *
+ * <p>The Remixer object is the heart and soul of Remixer as a framework, it coordinates syncing and
+ * value changes.
+ *
+ * <p>For value and data syncing/persistence (both locally across activities, and persisting/saving
+ * to cloud backends, etc) and keeping a global state, the Remixer object relies on a {@link
+ * SynchronizationMechanism} set at initialization time (usually in your Application.onCreate()).
```

Comment:
I just want to bring this advice to your attention. Is it possible that Application.onCreate() can be called more than once? Then remixer would crash since it is guarding in `setSynchronizationMechanism()`